README.md

```diff
@@ -105,6 +105,31 @@ Refer to the [documentation](https://docs.microsoft.com/dotnet/api/microsoft.sql
 
 **Note:** If you are replacing an existing `.sqlproj` be sure to copy over any of these properties into the new project file.
 
+## Model compiler options
+Like `.sqlproj` projects  `MSBuild.Sdk.SqlProj` supports controlling T-SQL build errors and warnings by using MSBuild properties..
+Treating warnings as errors can be optionally enabled by adding a property `TreatTSqlWarningsAsErrors` to the project file:
```

Comment:
I think we'll want to describe `TreatWarningsAsErrors` (ie. without `TSql`) here as well. Perhaps we can suffice with a note about this after this section, ie:

```
> Note: Alternatively one can use `TreatWarningsAsErrors` instead of `TreatTSqlWarningsAsErrors` to apply the same effect.
```